client/cluster-client/src/main/kotlin/uk/ac/ebi/biostd/client/cluster/api/SlurmClusterClient.kt

```diff
@@ -0,0 +1,119 @@
+package uk.ac.ebi.biostd.client.cluster.api
+
+import ebi.ac.uk.base.orFalse
+import ebi.ac.uk.coroutines.waitUntil
+import mu.KotlinLogging
+import uk.ac.ebi.biostd.client.cluster.common.JobSubmitFailException
+import uk.ac.ebi.biostd.client.cluster.common.toSlurmJob
+import uk.ac.ebi.biostd.client.cluster.model.Job
+import uk.ac.ebi.biostd.client.cluster.model.JobSpec
+import java.time.Duration.ofSeconds
+import kotlin.Result.Companion.failure
+import kotlin.Result.Companion.success
+
+private val logger = KotlinLogging.logger {}
+
+class SlurmClusterClient(
+    private val logsPath: String,
+    private val sshMachine: String,
```

Comment:
We should change this parameter to `sshServer` to make it consistent with the properties in the config file. WDYT?

Reply:
I have no particular stance on this so change looks good for me.